appveyor.yml

```diff
@@ -76,19 +78,15 @@
 
             if ($env:BOND_BUILD -eq "C++") {
 
-                if (!(Test-Path boost.7z)) {
-
-                    echo "Downloading Boost 1.${env:BOND_BOOST} ..."
-
-                    Start-FileDownload "http://iweb.dl.sourceforge.net/project/boost/boost/1.${env:BOND_BOOST}.0/boost_1_${env:BOND_BOOST}_0.7z" -FileName boost.7z -Timeout 1200000
-
+                if ($env:BOND_BOOST -eq 56) {
```

Comment:
Can we do this once for all the builds instead of having duplicate logic in two places?